src/pages/partners/_partnership-opportunities.js

```diff
@@ -92,20 +102,26 @@ const PartnershipOpportunities = () => (
                     )}
                 </Text>
             </ClientCard>
-            {/* TODO: add this when developer page is ready */}
-            {/* <ClientCard>
+            <ClientCard
+                to=""
+                is_deriv_developer_link
+                target="_blank"
+                external="true"
+                rel="noopener noreferrer"
+                third
+            >
                 <Flex ai="center" height="auto" mb="0.8rem">
                     <Header as="h4" size="var(--text-size-sm)">
-                        {localize('Developer programme')}
+                        {localize('API')}
                     </Header>
-                    <WithdrawIcon />
+                    <DeveloperProgramme />
                 </Flex>
                 <Text>
                     {localize(
-                        'Develop your own app or interface on top of our trading infrastructure. Earn affiliate commission from new signups, or mark up the price of the contracts traded in your app.',
+                        'Launch your trading app powered by our APIs. Get up to 5% in markup earnings from every trade your clients make on your app.',
```

Comment:
Do we need to make PR to crowdin instead since it has new string ?

Reply:
Nope we are releasing it straight to master first.